docs/src/index.md

```diff
@@ -17,26 +14,36 @@ Note that as the acceleration works by using the extents of geometries, the filt
 
 ## Usage
 Apply a spatial index on a Vector with geometries using `accelerate`, after which one can quickly find the intersecting extents for a given area of interest by `findall` with the `in` predicate.
+
 ```julia
-AA = accelerate(A, RTreeIndex)
-findall(in(aoi), AA)
+using GeoAcceleratedArrays
+using LibSpatialIndex
+AA = accelerate(A, SpatialIndex{Val{LibSpatialIndex}})
```

Comment:
Can we have a constructor that does this?

So `accelerate(A, SpatialIndex(LibSpatialIndex))` then dispatch on the object not the type

Reply:
AcceleratedArrays uses the type to eventually call `SpatialIndex{Val{LibSpatialIndex}}(A)`, so it takes a type, not an instance. But agreed it's ugly.

Reply:
Ok, maybe a function that generates the type?

`indextype(LibSpatialIndex)`